### ISSUE TITLE
Adjust default link color to increase color contrast

### ISSUE THREAD

**Issue by cmajel** [18F/brand#185]
In talking with @iamjolly, we discovered that the 18F slide template's default link color does not meet WCAG AA for small text (>18px, which happens often in our template).

Recommending that we switch to the `medium` shade in the 18F palette which meets these requirements, and also is more consistent with the identity site. 

## Current
Link color: #0097a7

![Screen Shot 2019-10-01 at 5 20 55 PM](https://user-images.githubusercontent.com/2374206/66001623-d8b8c000-e46f-11e9-94b7-6f4f22ab8b71.png)

## Proposed
Link color: #046B99 / `medium` in 18F palette
![Screen Shot 2019-10-01 at 5 23 19 PM](https://user-images.githubusercontent.com/2374206/66001764-31885880-e470-11e9-9814-d34140cd3c04.png)

**Comment by cmajel** [comment]
This has been updated 🎉
